### ISSUE TITLE
Questions about the KL divergence Loss

### ISSUE THREAD

**Issue by A1exy** [PuNeal/DLDL-v2-PyTorch#1]
Hi,  take a look at the KL loss code,  and  the second input of KL loss is labels,  I wondering it's different from the original paper. Do you have tried it on the age estimation dataset and reproduce the results of the paper more or less?

**Comment by PuNeal** [comment]
> Hi, take a look at the KL loss code, and the second input of KL loss is labels, I wondering it's different from the original paper. Do you have tried it on the age estimation dataset and reproduce the results of the paper more or less?

So happy that you focus on my work. In fact, the second input of KL Loss is the label distribution which is samled from normal distribution with mean equal to the real age of image. If you look into data.py and train.py, you will find that the second input of KL Loss is the label distribution.

**Comment by A1exy** [comment]
Thanks for your reply, I will try it later.
